tox.ini

```diff
@@ -2,13 +2,17 @@
 envlist =
     csslint
     eslint
-    py36
+    py{36}-django{111,20,21,22}
     pycodestyle
     pylint
 
 [testenv]
 deps =
-    coverage
+    -rrequirements/test.in
```

Comment:
We should add proper OEP-18 support later in a separate PR.

Reply:
Yeah, apologies; this was a bit quick-and-dirty to get some feedback in the form of tests.